packages/styles/src/core/lib/_reset.scss

```diff
@@ -1,155 +0,0 @@
-/* http://meyerweb.com/eric/tools/css/reset/
-   v2.0 | 20110126
-   License: none (public domain)
-*/
-
-/* apply a natural box layout model to all elements, but allowing components to change */
-html {
-  box-sizing: border-box;
```

Comment:
This is needed of everything is broken

Reply:
Also, I just realized we're adding it in `core.css`:

```
* {
  box-sizing: border-box;
}
```

Reply:
👍